kubeflow-pipelines/docling-vlm/docling_convert_components.py

```
@@ -0,0 +1,274 @@
+    from docling.datamodel.pipeline_options_vlm_model import ApiVlmOptions, ResponseFormat # pylint: disable=import-outside-toplevel  # noqa: PLC0415, E402
+
+    input_path_p = Path(input_path.path)
+    artifacts_path_p = Path(artifacts_path.path)
+    output_path_p = Path(output_path.path)
+    output_path_p.mkdir(parents=True, exist_ok=True)
+
+    input_pdfs = [input_path_p / name for name in pdf_filenames]
+
+    print(f"docling-convert: starting with backend='vlm', files={len(input_pdfs)}", flush=True)
+    if not pdf_filenames:
+        raise ValueError("pdf_filenames must be provided with the list of file names to process")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)